_code-samples/vaults/js/deposit.js

```diff
@@ -0,0 +1,135 @@
+// IMPORTANT: This example deposits into an existing PRIVATE vault.
+// The depositor account used has valid credentials in the vault's Permissioned Domain.
+// Without valid credentials, the VaultDeposit transaction will fail.
+// If you want to deposit into a public vault, you can replace the vaultID and shareMPTIssuanceId 
+// values with your own.
+
+import xrpl from "xrpl"
+
+// Connect to the network ----------------------
+const client = new xrpl.Client("wss://lend.devnet.rippletest.net:51233")
+await client.connect()
+
+const depositor = xrpl.Wallet.fromSeed("sEdVSq9Zsv8vQwfivTk37bWxrvpnruf")
+
+// The ID of the vault to deposit into
+const vaultID = "6AC4EC2D775C6275D314996D6ECDD16DCB9382A29FDB769951C42192FCED76EF" 
+// The ID of the vault's asset (MPT Issuance)
+const assetMPTIssuanceId = "0003E3B486D3DACD8BB468AB33793B9626BD894A92AB3AB4"
+// The ID of the vault's share (MPT Issuance)
+const shareMPTIssuanceId = "0000000152E7CD364F869E832EDB806C4A7AD8B3D0C151C5"
+
+console.log(`Depositor address: ${depositor.address}`)
+console.log(`Vault ID: ${vaultID}`)
+console.log(`Asset MPT issuance ID: ${assetMPTIssuanceId}`)
+console.log(`Vault share MPT issuance ID: ${shareMPTIssuanceId}`)
+
+const depositAmount = 10
```

Comment:
```suggestion
const depositAmount = 1
```
Reducing the amount here to reduce the rate we drain the prefunded MPT and hit insufficient balance error.